config/controller.yaml

```diff
@@ -28,9 +31,6 @@ spec:
     metadata:
       labels:
         app: net-contour-controller
-        app.kubernetes.io/component: net-contour
-        app.kubernetes.io/name: knative-serving
```

Comment:
Please allow me to double check. We don't need to keep these labels?
Other components like serving/controller has it https://github.com/knative/serving/blob/main/config/core/deployments/controller.yaml

Reply:
Good catch - I realize we'd probably want these labels on the Pod as well.

It may be worth creating a kourier issue then